weblate/templates/paginator.html

```diff
@@ -2,9 +2,83 @@
+     shortcut.add("alt+home", function() {
+
+       var a=document.createElement("a");
+       a.href="?page=1&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}" ;
+
+       document.body.appendChild(a);
+
+    });
+  </script>
+
+<script>
+ var prev_page=document.createElementById("prev_page");
+ shortcut.add("alt+left", function() {
+  var a=document.createElement("a");
+  a.href="?page={{ page_obj.previous_page_number }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}";
+
+  document.body.appendChild(a);
+})
+  </script>
+
+  <script>
+     var next_page=document.createElementById("next_page");
+ shortcut.add("alt+right", function() {
+  var a=document.createElement("a");
+  a.href="?page={{ page_obj.next_page_number }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}";
+
+  document.body.appendChild(a);
+})
+  </script>
+
+<script>
+  var next_page=document.createElementById("next_req_page");
+shortcut.add("alt+page_up", function() {
+var a=document.createElement("a");
+a.href="?page= {% blocktrans with page_obj.number as position and page_obj.paginator.num_pages as total %}{{ position }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}";
+
+document.body.appendChild(a);
+})
+</script>
+
+<script>
+  var next_page=document.createElementById("prev_req_page");
+shortcut.add("alt+page_down", function() {
+var a=document.createElement("a");
+a.href="?page= {% blocktrans with page_obj.number as position and page_obj.paginator.num_pages as total %}{{ position }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}";
+
+document.body.appendChild(a);
+})
+</script>
+
+
+
+<script>
+  var last_page=document.createElementById("last_page");
+shortcut.add("alt+end", function() {
+var a=document.createElement("a");
+a.href="?page={{ page_obj.paginator.num_pages }}&amp;limit={{ page_obj.paginator.per_page }}{% if page_obj.paginator.sort_by %}&amp;sort_by={{ page_obj.paginator.sort_by }}{% endif %}{% if query_string %}&amp;{{ query_string }}{% endif %}{% if anchor %}#{{ anchor }}{% endif %}";
+
+document.body.appendChild(a);
+})
+</script>
```

Comment:
* No inline JS (it will be blocked by Content-Security-Policy). 
* The URL can be obtained from the element and not generated again here.
* Please use Mousetrap to do the key bindings

Some keyboard shortcuts already exist, see:

https://github.com/WeblateOrg/weblate/blob/741a27950510d53e1fc2d0869acbbd4d429f87c4/weblate/static/editor/full.js#L39-L57